src/cli/mod.rs

```diff
@@ -584,45 +584,42 @@ fn log_result(
 }
 
 // Write a prompt to the terminal, and wait for an answer.
-fn prompt_message(message: &str, default: bool) -> bool {
-    let stdin = io::stdin();
-    let mut stdin = stdin.lock();
-
-    let stdout = io::stdout();
-    let mut stdout = stdout.lock();
-
-    let answer = &mut String::with_capacity(16);
-
-    enum Answer {
-        Continue,
-        Break(bool),
+fn prompt_message(message: &str, default: Option<bool>) -> bool {
+    let std_in = io::stdin();
+    let mut std_in_lock = std_in.lock();
+
+    let std_out = io::stdout();
+    let mut std_out_lock = std_out.lock();
+
+    fn get_input(
+        mut std_in: impl BufRead,
+        mut std_out: impl Write,
+        message: &str,
+    ) -> io::Result<String> {
+        print!("{}", message);
+        std_out.flush()?;
+
+        let mut buf = String::default();
+        std_in.read_line(&mut buf)?;
+        Ok(buf)
     }
 
-    let mut display_prompt = move || -> io::Result<Answer> {
-        answer.clear();
-
-        stdout.write_all(message.as_bytes())?;
```

Comment:
i've dropped the `bytes` handling. what is going to go wrong because i haven't done that?